filter_Extension.txt

```diff
@@ -1,3 +1,4 @@
+[uBlock Origin]
```

Comment:
Please change to `[AdGuard; uBlock Origin]`

Reply:
```suggestion
[AdGuard; uBlock Origin]
```

Reply:
Sorry, I saw the suggestion later